### PR TITLE
Clogged-scrubbers and false-alarm events

### PR DESCRIPTION
**Pull request by NanakoAC**
Porting an event from tg. Scrubbers get clogged and eject some chemical smoke with potentially fun effeects

I looked through chemsmoke code while doing this, and found it wanting, so i improved it. Added a duration input, adjusted all uses of chemsmoke in the code. Generally gave them all higher durations, chemsmoke grenades were a disappointing poot

Adds a new mundane event, false alarm.
It picks a random moderate or severe event and fakes its announcement without actually running the event.

Three minutes later, CC sends another announcement apologising for the false alarm

Also added an event var allowing events to exclude themselves from being picked for faking

### CODE REVIEW

File: code/modules/events/event_container.dm [hunk 1]
```diff
@@ -128,9 +128,9 @@ var/global/list/severity_to_string = list(EVENT_LEVEL_MUNDANE = "Mundane", EVENT
 	available_events = list(
 		// Severity level, event name, even type, base weight, role weights, one shot, min weight, max weight. Last two only used if set and non-zero
 		new /datum/event_meta(EVENT_LEVEL_MUNDANE, "Nothing",			/datum/event/nothing,			100),
-		new /datum/event_meta(EVENT_LEVEL_MUNDANE, "APC Damage",		/datum/event/apc_damage,		20, 	list(ASSIGNMENT_ENGINEER = 10)),
+		new /datum/event_meta(EVENT_LEVEL_MUNDANE, "APC Damage",		/datum/event/apc_damage,		25, 	list(ASSIGNMENT_ENGINEER = 10)),
 		new /datum/event_meta(EVENT_LEVEL_MUNDANE, "Brand Intelligence",/datum/event/brand_intelligence,20, 	list(ASSIGNMENT_JANITOR = 25),	1),
-		new /datum/event_meta(EVENT_LEVEL_MUNDANE, "Camera Damage",		/datum/event/camera_damage,		20, 	list(ASSIGNMENT_ENGINEER = 10)),
+		new /datum/event_meta(EVENT_LEVEL_MUNDANE, "Camera Damage",		/datum/event/camera_damage,		25, 	list(ASSIGNMENT_ENGINEER = 10)),
```
Comment (Arrow768) [inline code/modules/events/event_container.dm]: You have not indicated these changes in the description of the PR

Comment (NanakoAC) [inline code/modules/events/event_container.dm]: oh yea, just little tweaks, i already made a changelog entry for them in an earlier PR, should be going in together anyways

**Comment by skull132** [comment]
Holywater and coco sound like the weird ones on this list. Everything else is kinda fine, considering they're industrial chems/materials which can get in there.

Also, does flour smoke work?

**Comment by NanakoAC** [comment]
The list of chems was ported from tg, i just added a few, and removed some that don't exist in our codebase. I imagine holywater is for surprise fun of creating holy areas and hurting vampires.

As for coco, pretty sure thats just there for the color. the smoke takes on the color of the chems. That's the same reason banana is in there, the bright yellow.

And yes flour works, as does carbon and radium, they all create dirty floors in the area. Something unusual for the janitor to deal with

**Comment by skull132** [comment]
Merge error from #784 .

**Pull request closed without merging**